reaktive/src/commonMain/kotlin/com/badoo/reaktive/plugin/ReaktivePluginsExt.kt

```diff
@@ -0,0 +1,26 @@
+package com.badoo.reaktive.plugin
+
+import com.badoo.reaktive.annotations.ExperimentalReaktiveApi
+import com.badoo.reaktive.observable.Observable
+
+@ExperimentalReaktiveApi
+fun ReaktivePlugins.register(plugin: ReaktivePlugin) {
+    var list: MutableList<ReaktivePlugin>? = plugins
+    if (list == null) {
+        list = ArrayList()
+        plugins = list
+    }
+
+    list.add(plugin)
+}
+
+@ExperimentalReaktiveApi
+fun ReaktivePlugins.unregister(plugin: ReaktivePlugin) {
+    plugins?.remove(plugin)
```

Comment:
Check plugins size and null-out reference?

Reply:
Will do

Reply:
Implement it? We can avoid `fold` invocation on every call after unregistering all plugins.

Reply:
Oops, forgot about it! Thanks I will fix!